### ISSUE TITLE
Consecutive blank lines are forbidden should not apply inside template strings

### ISSUE THREAD

**Issue by corbinu** [palantir/tslint#1886]
### Bug Report

- __TSLint version__: `4.0.0`
- __TypeScript version__: `2.1.4`
- __Running TSLint via__: npm script

#### TypeScript code being linted

```ts
const str = `
    here is some text


`;
```

with `tslint.json` configuration:

```json
"rules": {
    "no-consecutive-blank-lines": [
        true
    ]
}
```

#### Actual behavior
Treats the consecutive blank lines inside the string as a violation

#### Expected behavior
Don't treat the consecutive blank lines inside the string as a violation

**Comment by adidahiya** [comment]
sounds like a bug, accepting PRs

**Comment by IllusionMH** [comment]
Is there any alternatives to `getTokenAtPosition` function?
There is easy fix with
```ts
const token = (ts as any).getTokenAtPosition(node, start) as ts.Node;
```
but it is not a part of public API (however `ts as any` already used in [noInferredEmptyObjectTypeRule.ts#L55](https://github.com/palantir/tslint/blob/master/src/rules/noInferredEmptyObjectTypeRule.ts#L55) )

I tried to use `LanguageService` but none of methods that has `AtPosition` returns needed info.

**Comment by nchen63** [comment]
the `ts as any` was only used there to handle backcompat with TypeScript v2.0, since the field doesn't exist in Typescript v2.1.

Seems strange that `getTokenAtPosition` is not public. I don't think we should be using private functions. I guess the alternative would be to walk every node to find the template string nodes, then filter out the errors

**Comment by ajafff** [comment]
I have a fix in my local fork. Will prepare a PR tomorrow(ish)
